typedspark/_schema/get_schema_definition.py

```diff
@@ -59,7 +62,17 @@ def _build_schema_definition_string(
             typehint, replace_literals_in=DayTimeIntervalType, replace_literals_by=IntervalType
         )
         if include_documentation:
-            lines += f'    {k}: Annotated[{typehint}, ColumnMeta(comment="")]\n'
+            if k in schema.__annotations__:
+                if (
+                    hasattr(schema.__annotations__[k], "__metadata__")
+                    and schema.__annotations__[k].__metadata__ is not None
+                ):
+                    lines += (
```

Comment:
Here and in the part below the piece `f"    {k}: Annotated[{typehint}, "` is repeated. Alternatively, you could also set the comment value in the if/else block and add to lines outside the if/else block